assignment6.rb

```diff
@@ -0,0 +1,62 @@
+class Assignment
+    def eliminate_nil(arr)
+       print arr.compact()
```

Comment:
I told you you should not use any pre-built ruby methods. Please write the logic to remove nil values.